frontend/src/components/NavBar.tsx

```diff
@@ -116,11 +137,11 @@ export default function NavBar() {
               open={Boolean(anchorElUser)}
               onClose={handleCloseUserMenu}
             >
-              {settings.map((setting) => (
```

Comment:
Do you think keepMounted property should be used here ?

Reply:
it is used for the intend to enhance user experience by maintaining the menu's state between open and close actions. If we want the menu to stay mounted even when it's closed, we should include the keepMounted prop. I used MUI component and seaerched on later about it.